tests/subsys/fs/fat_fs_api/src/test_fat_file.c

```diff
@@ -38,7 +38,411 @@ static int test_file_open(void)
 		TC_PRINT("File now exists %s\n", TEST_FILE);
 	}
 
-	return res;
+	return TC_PASS;
+}
+
+static int test_file_open_flags(void)
+{
+	int res;
+	struct fs_file_t fp;
+	char something[] = "Something";
+	char any[] = " Any";
+	char anything[] = " Anything";
+	char buffer[sizeof(something) + 1] = { 0 };
+
+	TC_PRINT("\nfs_open flags tests\n");
+
+	TC_PRINT("* Attempt to open non-existing file (expect fail)\n");
+	/* Should fail */
+	res = fs_open(&fp, FATFS_MNTP"/missing", 0);
```

Comment:
IMO it's easier to read these if there's whitespace between the macro token and the string.  Though if this is the style used in this file, I guess we live with it (or change the existing uses in a separate whitespace-only commit).

Reply:
I am reworking tests into more readable form now, so do not bother looking at them for now.

Reply:
Done.